ivadomed/transforms.py

```diff
@@ -278,6 +281,102 @@ def __call__(self, sample, metadata=None):
         return data_out, metadata
 
 
+class Resize(ImedTransform):
```

Comment:
isn't there already the [Resample()](https://github.com/ivadomed/ivadomed/blob/master/ivadomed/transforms.py#L211) method that you could use as per @andreanne-lemay comment https://github.com/ivadomed/ivadomed/issues/766#issuecomment-831566233?

Reply:
The Resample method works on mm while we have pixel-level data. Besides that, with resize we can have an interpolation option.

Reply:
> The Resample method works on mm while we have pixel-level data.

It should be relatively straightforward to slightly modify the `Resample()` method to accommodate pixel-wise dimensions, instead of duplicating code. We can help if you don't know how to do this.

Besides that, with resize we can have an interpolation option.

Reply:
The `Resample()` class uses the `scipy.ndimage.zoom` method, which scales the image based on the given factor. The scale factor has no relation to millimeters, pixel or et. al, and it is based on the ratio of the input sample's scale. Initially based on the `mm `comment in the class description I thought it only works for millimeters but actually, it is based on input scale. It is better to use input scale in class description rather than `mm` to be more clear. 
Using constant scale factor (based on constant value from the config file) can produce variable sizes for inputs with different sizes, hence, I used input size to calculate the desired factors. 
Class `Resize `is removed now.